### DISCUSSION TITLE
Metaplex Whitepaper NFT Incentive Program

### DISCUSSION THREAD

**Discussion by svc01-metaplex** [metaplex-foundation/mip#34] [MIP Submissions]
Submitter: Charlesfoodog
Email: charlessublette@gmail.com
Submitter wallet: 2YxHRtQJ914rQkxqyFYC1VxfpJWMZHLqc7SAMPVjipRv
Submission transaction: 4JKCR1sadQ89YxGhZBB7oQRbsnoYB4MU5HpNxqMvMaB2xuwWL9PXe8aujf6fFfpkRjMycrTucn9iEMNEdrE5hMZS
Title: Metaplex Whitepaper NFT Incentive Program 
Links: https://magiceden.io/marketplace/metaplex_whitepaper

This proposal is to allocate 24,000,000 Metaplex tokens from the Metaplex DAO  treasury to Metaplex Whitepaper Holders.  24,000 tokens with a vesting of 2000 MPLX tokens per NFT holder (1000 NFTs exist in the collection in total)  for 24 months.   Metaplex NFT holders will get access to a special exclusive chat section in the Metaplex discord.  

Starting distributions will begin June 1st, 2024, and will end June 1st, 2026. Each month a snapshot will be taken of current holders. This will occur at a randomly picked time inside each monthly block. Distributions will also be at the end of every month.

These tokens distribution will be to incentivize community growth, community, and engagement inside the Metaplex ecosystem.  As well add another level of decentralization of the token.

**Comment by MetaplexFoundation** [comment]
Thanks for the submission, Charlesfoodog. MIP submissions are limited to proposals for changes to the Metaplex Program Library. Please see here for the MIP guidelines (https://github.com/metaplex-foundation/mip/blob/main/mip-5.md). Accordingly, we are screening out this MIP submission and closing the discussion. 

Requests for allocations of tokens from the Metaplex DAO treasury should be made via MPLX grant proposals. To submit an MPLX grant proposal, please see here: https://mip.metaplex.com/
